src/models/transfers.js

```diff
@@ -461,10 +453,26 @@ function * getFulfillment (transferId) {
   return fulfillment.getDataExternal()
 }
 
+function * resendNotification (transferId) {
```

Comment:
That could be a good feature. Let's do that when we have a clear use case as why this would be required. That will be easy to add later.